features/nationaliteithistorie.feature

```diff
@@ -21,7 +21,7 @@ Functionaliteit: Tonen van Nationaliteithistorie
   Een beëindigde nationaliteit waarbij de jongste bijbehorende historische categorie 54 met nationaliteit (05.10) gevuld én indicatie onjuist (84.10) gevuld, wordt niet opgenomen in het antwoord.
   Voor een actuele nationaliteit (niet-beëindigd) met een bijbehorende historische categorie 54 met indicatie onjuist, worden de gegevens (incl. datum ingang) in de onjuiste categorie genegeerd.
 
-  In het antwoord worden de eerst de actuele nationaliteiten opgenomen, gevolgd door de beëindigde nationaliteiten (gesorteerd op indicatieNationaliteitBeeindigd). Daarbinnen worden resultaten aflopend gesorteerd op datumEindeGeldigheid en vervolgens aflopend gesorteerd op datumIngangGeldigheid.
+  In het antwoord worden de eerst de actuele nationaliteiten opgenomen, gevolgd door de beëindigde nationaliteiten (gesorteerd op indicatieNationaliteitBeeindigd). Daarbinnen worden resultaten aflopend gesorteerd op datumTot en vervolgens aflopend gesorteerd op datumIngangGeldigheid.
```

Comment:
Wijzig 'In het antwoord worden de eerst' in 'In het antwoord worden eerst'.

Reply:
In regel 8 van dit document staat 

'Voor een beëindigde nationaliteit of beëindigd bijzonder Nederlanderschap worden de eigenschappen overgenomen uit de jongste bijbehorende historische categorie (54) waarin deze zijn voorkomen.'

Volgens mij moet het op een na laatste woordje ('zijn') worden verwijderd.

Reply:
Aangepast

Reply:
Johan,

Je geeft aan dat je bovenstaande opmerkingen van mij hebt aangepast maar ik zie die aanpassingen niet.